frontend/src/app/utils/testResultCSV.ts

```diff
@@ -1,115 +1,149 @@
 import moment from "moment";
 
-import {
-  byDateTested,
-  Results,
-} from "../testResults/viewResults/TestResultsList";
-import { TEST_RESULT_DESCRIPTIONS } from "../constants";
+import { byDateTested } from "../testResults/viewResults/TestResultsList";
 import { MULTIPLEX_DISEASES } from "../testResults/constants";
+import { GetFacilityResultsForCsvWithCountQuery } from "../../generated/graphql";
 
 import { hasSymptomsForView, symptomsStringToArray } from "./symptoms";
-import { getResultByDiseaseName } from "./testResults";
 
 import { displayFullName, facilityDisplayName } from "./index";
 
+export type QueriedTestResult = NonNullable<
+  NonNullable<
+    GetFacilityResultsForCsvWithCountQuery["testResultsPage"]
+  >["content"]
+>[number];
+
+export interface ResultCsvRow {
+  "Patient first name": string | null | undefined;
```

Comment:
Any chance we could define this interface by taking the CSV headers as keys and then using mapped types? Something like 

```ts
const resultCSVHeadersString = [
  "Patient first name",
  "Patient middle name",
  ...
] as const

const resultCSVHeadersBoolean = [
  "Patient is a resident in a congregate setting",
  "Patient is employed in healthcare",
] as const;

type ResultCsvRow = {
  [K in typeof resultCSVHeadersBoolean[number]]: boolean | null | undefined
} | {
  [K in typeof resultCSVHeadersString[number]]: string | null | undefined
}
```
That way if anyone ever needs the CSV headers in the future, they can just pull the initial constants out and use them independently of the type